pubspec.yaml

```diff
@@ -53,6 +53,7 @@ dependencies:
 
   # An addition to the material icons.
   ionicons: ^0.2.1
+  intl: ^0.18.1
```

Comment:
I am planning to use `intl` in the future, but I would like to do that when I rework the whole project to use it. It doesn't seem so necessary as of now, so could you remove it?

Reply:
I was currently only using this package to get the Weekday from a date, however I've removed this now and changed it with a method that does not use `intl`.